### ISSUE TITLE
CUDA/OpenCL testing some algos not working AMD/nVidia

### ISSUE THREAD

**Issue by Xn0xic** [MoneroOcean/xmrig#148]
nVidia 3000 series- cn/gpu, panthera, ghostrider, flex, argon2/chukwav2: Algo skipped due to disabled backend

AMD 5000 and 6000 series- panthera, flex, argon2/chukwav2: Algo skipped due to disabled backend                                              

 - XMRig version 6.21.3-mo15
    - https://github.com/MoneroOcean/xmrig/releases/download/v6.21.3-mo15/xmrig-v6.21.3-mo15-win64.zip
    
 - Miner log as text or screenshot
 
![NVD](https://github.com/MoneroOcean/xmrig/assets/133929080/0aadf14e-3c1b-4f5e-b087-335b7af7eb61)
![AMD](https://github.com/MoneroOcean/xmrig/assets/133929080/b443a1f2-644b-463d-b12f-26befd317430)

 OS: Win 11 Pro
 - For GPU related issues: 3080ti/3070ti 555.85/CUDA 11.4 and 12.5.  RX5500XT/5700XT/RX6700XT/6750XT/6800XT Adrenalin 24.3.1

**Additional context**
Thanks for all the work you have been putting in gents. :)

**Comment by Spudz76** [comment]
* cn/gpu should work in all cases, perhaps you are using xmrig-cuda vanilla from upstream? (must use xmrig-cuda MO-fork)
* panthera/ghostrider/flex/argon2 simply don't have GPU support at all, normal and expected

might be worth writing ghostrider/flex/argon2 support but that is unclear

panthera is RandomX and none of those, even the supported ones, are worth it

**Comment by Xn0xic** [comment]
Confirming I'm using the xmrig-cuda MO-fork from here: https://github.com/MoneroOcean/xmrig-cuda/releases/download/v6.21.1-mo1/xmrig-cuda-v6.21.1-mo1-cuda11_4-win64.zip

Looking back at previous config & log files panthera/argon2/ghostrider all ran for a benchmark during initial setup, but never ran (or even attempted) after the initial run.

"worth it" is subjective, but I do concede that in terms of H/W it is horribly inefficient and usually not a profitable venture.  There was a hot minute there in Oct-Dec 2023 where it was actually the most profitable to run rx/0 for ZEPH on GPUs.  Right now, kawpow is where it's at, but my GPUs do flip to autolykos2 and CN algos frequently enough that I also using mm.exe to switch between XMRig for rx/ and cn/ and GPU specific miners for kawpow and autolykos2.

**Comment by Spudz76** [comment]
Yeah same, but I run a GTX1060 so RandomX is absolutely never worth it H/W wise.  Kawpow is barely even actually worth it since I get ~5KH/s pay rate for 140W which is even worse than my crappy old Sandybridge CPUs.

**Comment by Xn0xic** [comment]
Ouch.  6700XT with kawpow caps at about 140W for 13ish kh/s equivalent to rx/0 right now. 5500XT is 58W for 6kh/s equivalent.  Those are my two most efficient cards.  nVidia 3000 series cards are 25% less efficient.

**Comment by hajes** [comment]
xmrig-cuda plugin on MO github redirects to official xmrig-cuda website
